examples/quirrel/app/routes/index.tsx

```diff
@@ -0,0 +1,41 @@
+import type { LoaderFunction } from "remix";
+
+import greetingsQueue from "~/queues/greetings.server";
+
+export const loader: LoaderFunction = async () => {
+	await greetingsQueue.enqueue("Groot");
+	return null;
+};
+
+export default function Index() {
+	return (
+		<div style={{ fontFamily: "system-ui, sans-serif", lineHeight: "1.4" }}>
+			<h1>Welcome to Remix</h1>
+			<ul>
+				<li>
+					<a
+						target="_blank"
+						href="https://remix.run/tutorials/blog"
+						rel="noreferrer"
+					>
+						15m Quickstart Blog Tutorial
+					</a>
+				</li>
+				<li>
+					<a
+						target="_blank"
+						href="https://remix.run/tutorials/jokes"
+						rel="noreferrer"
+					>
+						Deep Dive Jokes App Tutorial
+					</a>
+				</li>
+				<li>
+					<a target="_blank" href="https://remix.run/docs" rel="noreferrer">
+						Remix Docs
+					</a>
+				</li>
+			</ul>
+		</div>
+	);
```

Comment:
This could be simplified. If there's nothing UI relevant in this example then you can just say so in one line:

```suggestion
	return <p>Check the code. Nothing relevant in the UI.</p>;
```